kubejs/assets/kubejs/lang/en_us.json

```diff
@@ -107,6 +107,11 @@
     "item.kubejs.basic_circuit_package": "Basic Control Circuit Assembly Package",
     "item.kubejs.basic_circuit_assembly": "Unpacked Basic Control Circuit Assembly Package",
 
+    "item.kubejs.superheated_steel_ingot": "Superheated Steel Ingot",
+    "block.kubejs.superheated_steel_block": "Superheated Block of Steel",
+    "item.kubejs.hot_compressed_iron_ingot": "Hot Compressed Iron Ingot",
+    "block.kubejs.hot_compressed_iron_block": "Hot Compressed Iron Block",

```

Comment:
Should be `Hot Block of Compressed Iron` to match naming conventions for blocks of metal :)